packages/dds/sequence/src/defaultMapInterfaces.ts

```diff
@@ -222,10 +224,10 @@ export interface IValueTypeOperationValue {
 	/**
 	 * The name of the operation.
 	 */
-	opName: string;
+	opName: IntervalOpType;
 
 	/**
 	 * The payload that is submitted along with the operation.
 	 */
-	value: any;
+	value: SerializedIntervalDelta | undefined;
 }
```

Comment:
this is the main change, all other change flow from this, as this couples the value type envelope to the interval collection type.